utils/user_info.py

```diff
@@ -191,3 +193,27 @@ def send_forget_password_email(request, user):
         Thread(target=msg.send, args=()).start()
     except:
         logger.warning("There is an error when sending email to %s's mailbox" % username)
+
+def get_public_users():
+    return User.objects.filter(username__startswith = ANONYMOUS_PREFIX)
+
+def attends_not_ended_contest(user):
+    user_attends = Contestant.objects.filter(user = user)
+    for contestant in user_attends:
```

Comment:
Since it's a Contestant object, I think contestant will be better. ( But I still think the name is not clear enough